src/patternfly/components/Page/page.scss

```
@@ -76,8 +76,9 @@ $pf-page-v6--height-breakpoint-map: build-breakpoint-map("base", "sm", "md", "lg
   --#{$page}__main-container--MarginInlineStart: var(--#{$page}--inset);
   --#{$page}__main-container--MarginInlineEnd: var(--#{$page}--inset);
   --#{$page}__main-container--BorderRadius: var(--pf-t--global--border--radius--medium);
-  --#{$page}__main-container--BorderWidth: #{pf-size-prem(4px)}; // TODO Change to be a page outline token
-  --#{$page}__main-container--BorderColor: var(--#{$page}__main-container--BackgroundColor); // TODO Border should match the background to blend in - change to be a page outline token
+  --#{$page}__main-container--BorderWidth: var(--pf-t--global--border--width--page--default);
+  --#{$page}__main-container--BorderColor: var(--pf-t--global--border--color--page--default); 
+  --#{$page}__main-container--Padding--offset: 4px;
```

```suggestion
  --#{$page}__main-container--BorderWidth: var(--pf-t--global--border--width--page--default);
  --#{$page}__main-container--BorderColor: var(--pf-t--global--border--color--page--default); 
```

I think that we can just make this change and revert all the other changes to the file. I don't think we need the offset because subtracting the border width from the page section(s) padding (which was already done) will give the correct spacing and avoid the 2px gap between the high-contrast border and a section with a secondary background. There will be a slight vertical shift as the top border goes from 4 to 2px, but I think that's acceptable as we currently don't try to vertically align the main area with anything else.

If we want to avoid the vertical shift, we can select the first page group/section and adjust, but it's a bit messy and perhaps not worth it.

@lboehling @mcoker

---

i'm good with this 👍 thank you!

---

follow up Q -- we had discussed updating `--pf-t--global--border--width--page--default` & `--pf-t--global--border--color--page--default` to use "main" instead of "page" do we still want to do that?

---

I don't hate the idea of that 😄 But I don't consider it a blocker to this.

---

I like it. We could then use page areas as names - "header", "sidebar", possibly "nav" if needed, and "main". Not a blocker to this PR, which I'd like to get merged ASAP but we'd want to make that change this sprint before code freeze.

---

sounds good -- i'll make that change in figma so it's ready the next time we export the tokens.